### ISSUE TITLE
Does Training support Chunking?

### ISSUE THREAD

**Issue by gofreelee** [aqlaboratory/openfold#165]
Hello, I look forward to observing the dynamics of the network during training. I noticed that Chunking is turned off during training, and there is related assert in code, so is it not supported to start Chunking during training?

**Comment by gofreelee** [comment]
Is it true that during training, no matter whether the input size changes or not, it always goes through the same calculation path？thanks in advance

**Comment by gahdritz** [comment]
During training, input proteins are randomly cropped to a certain fixed length. Proteins shorter than the crop size are padded.

**Comment by gofreelee** [comment]
So can I assume that during training, all training data have the same computational logic?

**Comment by gahdritz** [comment]
Yes, I think so.

**Comment by pengzhangzhi** [comment]
I am wondering if your code support batch training? backprop and update the model based on the gradients on multiple proteins?

**Comment by gahdritz** [comment]
We do support batch training. See the `batch_size` option in the config. The peak memory usage of the model is high enough that it's often impractical to increase it past 1, but for different training data or smaller crops than those used during model finetuning it might make sense.
